source/dotnet/Library/AdaptiveCards/Rendering/InputsLabelConfig.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace AdaptiveCards.Rendering
```

Comment:
Naming: InputLabelConfig and InputLabelsConfig as referring to the label configuration and label collection configuration seems pretty confusing.  What's the point of having these split into 2 separate classes again? let me know if you discuss this elsewhere in the spec wherei should be looking..